api_tests/lib/config.ts

```diff
@@ -20,7 +20,10 @@ export class E2ETestActorConfig {
     constructor(
         public readonly httpApiPort: number,
         public readonly comitPort: number,
-        public readonly name: string
+        public readonly name: string,
+        public readonly lndP2pPort: number,
+        public readonly lndRpcPort: number,
+        public readonly lndRestPort: number
```

Comment:
This is not needed.

Reply:
Why?  If we don't have these then LND defaults to its own default ports - these will then clash between the two instances, let alone when we try to parallelise the tests.

Reply:
So let's deactivate it then,

Reply:
Oh, you are talking about just the rest port.  You are right it isn't needed.